src/core/input.rs

```diff
@@ -59,7 +59,7 @@ pub struct InputDependency {
     pub missing: bool,
     pub extraneous: bool,
     pub development: bool,
-    pub version: String, // on disk
+    pub version: String,             // on disk
```

Comment:
wtf what options do you have for rustfmt? should we align this?

Reply:
tempted to "just do what the formatter says" on this.